### PR TITLE
Project Structure Updated

### PR DESCRIPTION
**Pull request by hntd187**
1. Moved all projects to a standard src/main/scala and src/test/scala in order to let us use the new java API
2. Upgraded the build to sbt 0.13.12, which .scala builds are deprecated since .sbt builds have come to be basically the same thing. The new builds are basically the same thing at this point
3. ooyala package changed to spark.jobserver
4. Netty was upgraded to be the same as the Cassandra driver, so the Cassandra driver is more consistent.

With this up, I should have the java api followed soon.

### CODE REVIEW

**Comment by velvia** [comment]
Awesome thanks.  Let me see about merging the other small PRs first.

File: job-server-python/src/python/test/apitests.py [hunk 1]
```diff
@@ -42,7 +43,7 @@ def validate(self, context, runtime, config):
 
     def run_job(self, context, runtime, data):
         rdd = context._sc.parallelize(data)
-        df = context.createDataFrame(rdd, ['name', 'age', 'salary'])
+        df = context.createDataFrame(data, ['name', 'age', 'salary'])
```
Comment (zeitos) [inline job-server-python/src/python/test/apitests.py]: I think you can remove the previous line right?

File: job-server-python/src/python/test/apitests.py [hunk 2]
```diff
@@ -56,6 +57,8 @@ class TestSJSApi(unittest.TestCase):
     def setUp(self):
         conf = SparkConf().setAppName('test').setMaster('local[*]')
         self.sc = SparkContext(conf=conf)
+        self.jvm = self.sc._gateway.jvm
+        java_import(self.jvm, "org.apache.spark.sql.*")
```
Comment (zeitos) [inline job-server-python/src/python/test/apitests.py]: spark sql is mandatory ?

Comment (hntd187) [inline job-server-python/src/python/test/apitests.py]: @zeitos hmm, I thought I backed out all those changes, but guess not. I'll update to take that out, I'm not sure what went wrong there.

**Comment by hntd187** [comment]
@velvia have we merged the pull requests you wanted? Can I merge this?

**Comment by velvia** [comment]
@hntd187 you know why don't we just merge it.  I was hoping to take care of 
https://github.com/spark-jobserver/spark-jobserver/pull/545
and #623 

but I think we are waiting on the original posters.  :/

**Comment by zeitos** [comment]
@hntd187 looks good to me stephen!

**Comment by velvia** [comment]
+1  Thanks!

**Pull request merged**